packages/design-tokens/design-token-previews/utils.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import map from "lodash/map";
+import keys from "lodash/keys";
+import camelCase from "lodash/camelCase";
+import capitalize from "lodash/capitalize";
+import invokeMap from "lodash/invokeMap";
+
+import fontSizeTokens from "../src/tokens/font-size.tokens.json";
+import fontWeightTokens from "../src/tokens/font-weight.tokens.json";
+import colorTokens from "../src/tokens/color.tokens.json";
+import iconSizeTokens from "../src/tokens/size.tokens.json";
+import spaceTokens from "../src/tokens/space.tokens.json";
+import borderStyleTokens from "../src/tokens/border-style.tokens.json";
+import borderWidthTokens from "../src/tokens/border-width.tokens.json";
+import borderRadiiTokens from "../src/tokens/border-radius.tokens.json";
+
+import {
+  createBorderRadiiPreview,
+  createSpacePreview,
+  createColorPreview,
+  createFontSizePreview,
+  createIconSizePreview,
+  createFontWeightPreview,
+  createBorderStylePreview,
+  createBorderWidthPreview,
+} from "./utils/previews";
+
+import { hexToHsla, hexToRgb } from "./utils/colors";
+import { Token, TokenEntry } from "./types/Token";
+import { Column, Row } from "./types/TokenTable";
+
+import { CopyToClipboardButton } from "./CopyToClipboardButton";
+
+const getKey = (object: Record<string, unknown>): string => {
+  return keys(object)[0];
+};
+
+const getValue = ([, token]: TokenEntry): string => {
+  return token.value;
+};
+
+const getComment = ([, token]: TokenEntry): string => {
+  return token.comment || "";
+};
+
+const formatTokenName = (prefix: string, suffix: string) => {
+  const tokenName = camelCase(`${prefix}${capitalize(suffix)}`);
+  return (
+    <div style={{ display: "flex" }}>
+      <div>{tokenName}</div>
+      <div>
+        <CopyToClipboardButton textToCopy={tokenName} />
+      </div>
+    </div>
+  );
+};
+
+export const TOKENS = {
```

Comment:
Shouldn't we keep it in a constants file?

Reply:
I also think we should rethink this variable name. It isn't about tokens but columns or transformations.